app/code/Magento/InventoryApi/Test/_files/product_configurable.php

```diff
@@ -0,0 +1,148 @@
+<?php
```

Comment:
Need to move all of logic to `InventoryConfigurableProduct`

Reply:
Done.